<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTeam</class>
 <widget class="QWidget" name="NewTeam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Team</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>New Team</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNewTeamID">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Team ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Team Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCreateTeam">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Team</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNewTeamName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ediSolo.qrc">:/new/prefix1/ediSolo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="oopsName" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QLabel" name="bubbleName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ediSolo.qrc">:/new/prefix1/rBubble.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_oopsName">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>7</y>
      <width>106</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Oops!
Team Name can be up to 30 characters.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_oopsName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: none;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="oopsId" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QLabel" name="bubbleId">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ediSolo.qrc">:/new/prefix1/rBubble.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_oopsId">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>7</y>
      <width>106</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Oops!
Team ID can be up to 5 characters.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_oopsID">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: none;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="oopsIdOrName" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QLabel" name="bubbleIdOrName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ediSolo.qrc">:/new/prefix1/rBubble.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_oopsIdOrName">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>8</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Oops!
Team ID or Name already in use.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_oopsIdOrName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: none;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="oopsMissingFields" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QLabel" name="bubbleMissingFields">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ediSolo.qrc">:/new/prefix1/rBubble.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_MissingFields">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>7</y>
      <width>106</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Oops!
Missing fields.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MissingFields">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: none;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ediSolo.qrc"/>
 </resources>
 <connections/>
</ui>
